src/main/webapp/resources/js/pages/analysis/components/outputs/OutputFilePreview.jsx

```diff
@@ -152,7 +177,9 @@ export default function OutputFilePreview() {
       if (
         !jsonExtSet.has(output.fileExt) &&
         !tabExtSet.has(output.fileExt) &&
-        !blacklistExtSet.has(output.fileExt)
```

Comment:
Isn't this a duplicate set of code from AnalysisOutputsContext?

Reply:
In AnalysisOutputsContext we use this to check if there is a file with type `extension` within the outputs. In the OutputFilePreview we use it to create the preview for that type of file. It is run within a for loop so we need to check if the current output from the array is that type

Reply:
Wouldn't it make more sense to have the context return a function called `getPreviewForFileType`  and have that stored when you first iterate over all the types?

Reply:
Yup totally does. Updated in a6f973b9319bc3f2d954a416af2062bb5b776ba8